docs/q1_report.md

```diff
@@ -0,0 +1,81 @@
+# Operate First Data Science Community Meetup Analytics Report: Q1Y22
+
+The following report outlines some key metrics and associated analysis of the community engagement for the Operate First Data Science Community Meetup between November 16, 2021 and March 31, 2022. If you would like to view recording of past meetups, or see the complete analytics dashboard yourself, please use the following to links: 
+
+* [Detailed dashboard](https://datastudio.google.com/u/1/reporting/de85ebeb-c76a-4930-85bf-b07a1d2becd3/page/qiqpB/edit)
+* [Past meetups and recordings](content.md)
+
+
+### Aim of the community 
+
+The Operate First Data Science Community Meetup is an event open to everyone interested in learning more about how various individuals, teams, and online communities are working towards democratizing the development of production-level AI. 
+
+This meetup aims to embrace the multi-disciplinary nature of the current state of AI operations and cloud infrastructure by covering technical topics relevant to data scientists, software engineers, DevOps professionals, and statisticians alike. And all in an open public forum.
+
+
+### Number of Meetups hosted 
+
+8
+
+### Total Number of Attendees
+
+155
+
+### Total Number of Unique Attendees
+
+74
+
+### Average number of Attendees per Meetup
+
+28
+
+
+### Internal vs external attendees
+
+
+![image 1](assets/image1.png)
+
+### Number of attendees for each meetup 
+
+**Consistent attendance across meetups with the lowest number of attendees being 21**
+
+![image 2](assets/image2.png)
+
+### Speakers from various teams
+
+**So far we have had **one external speaker** from the OSPO team present at the meetup**
+
+![image 3](assets/image3.png)
+
+
+## Meetup Engagement on YouTube
+
+To date we have recorded and posted 8 videos of the Operate First Data Science Community Meetup to our YouTube channel. 
+
+### Shares on Meetup Videos posted on Youtube
+
+**3 videos from the 8 operate first DS meetup videos are in the top 10 shared videos across the channel**
+
+
+![image 4](assets/image4.png)
+
+
+### Youtube Channel Subscribers obtained through Meetup videos.
+
+
+**Operate First Community Meetup Videos accounted for over 30% of the channel’s subscribers**
+
+![image 5](assets/image5.png)
+
+
+### Views 
+
+**4th highest viewed video across the channel is Operate First Community Meetup #1 amounting to 193 views**
```

Comment:
"#1" links to the first pull request in the markdown, I guess because that's how github works.

Reply:
Thanks, fixed this!